.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,67 @@
+name: 🐞 Bug Report
+description: File a bug report
+title: "[Bug]: <title>"
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+        
+        The following fields are required:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [Code of Conduct](https://github.com/enarx/.github/blob/main/CODE_OF_CONDUCT.md)
+      options:
+        - label: I agree to follow this project's Code of Conduct
+          required: true
+  - type: textarea
+    attributes:
+      label: Current Behavior
+      description: A concise description of what you're experiencing.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: A concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Environment Information
+      description: "Please paste output of following command: `cat /etc/*-release ; enarx --version; enarx info; lscpu`"
```

How about something like this instead?
- `uname -a` should give us a bit more useful data, i.e. kernel version and distro
- we only really need the CPU model and flags, so we can avoid dependency on `lscpu` and increase portability
```suggestion
      description: "Please paste output of following command: `uname -a; enarx --version; enarx info; cat /proc/cpuinfo | grep -m 2 -E 'model name|flags'`"
```

I think we don't actually need to query CPU data manually, @npmccallum @haraldh I remember seeing some kind of tool, which I think we maintain ourselves handling all of this? I think @npmccallum used it to check SGX support.
In any case, we should provide a subcommand on `enarx` in the long run for collecting all the relevant diagnostics and pasting the output here.

---

I think in the long run a diagnostic subcommand would be great. Proposed change applied.